cmd/controller-manager/app/controllermanager.go

```diff
@@ -206,7 +206,7 @@ func Run(ctx context.Context, opts *options.Options) error {
 var controllers = make(controllerscontext.Initializers)
 
 // controllersDisabledByDefault is the set of controllers which is disabled by default
-var controllersDisabledByDefault = sets.New("hpaReplicasSyncer")
+var controllersDisabledByDefault = sets.New("hpaAutoLabelRetain", "deploymentReplicasSyncer")
```

Comment:
```suggestion
var controllersDisabledByDefault = sets.New("hpaScaleTargetMarker", "deploymentReplicasSyncer")
```
How about taking the `hpaScaleTargetMarker` name?
In my opinion, the controller is responsible for marking objects that scaled by a HPA. 
The controller uses a `label` to mark these objects, how this label will be applied, it's irrelevant to this controller.

@XiShanYongYe-Chang What do you think?

Reply:
👍